package.json

```diff
@@ -29,7 +29,7 @@
     "scripts/*"
   ],
   "scripts": {
-    "build": "nx run-many --target=build --nx-ignore-cycles",
+    "build": "nx run-many --target=build --nx-ignore-cycles --skip-nx-cache",
```

Comment:
This makes sense while we're working on this but do you think we should make sure to remove it before it gets merged?

Reply:
woops, this should not be in here , good catch!!